extensions/amp-story/0.1/amp-story.css

```diff
@@ -268,24 +268,23 @@ amp-story:not([desktop]) >
   z-index: 2 !important;
 }
 
+.i-amphtml-story-layer, amp-story-cta-layer, amp-story-grid-layer {
```

Comment:
Actually, the more I look at this, the more I think I'm in favor of not including `.i-amphtml-story-layer` in this selector.  It's by *coincidence* that all `.i-amphtml-story-layer` implementations have these styles in common, not by *definition*, so I think I'd be more in favor of applying it to the layer implementations themselves (which, by definition, have these styles).

Reply:
Makes sense 👍